src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -29,6 +29,7 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
   const { id: venueId, isLive: isPosterLive, poster, iframeUrl } = venue;
 
   const { title, introVideoUrl, categories } = poster ?? {};
+  const number = venue.name.replace("poster", "");
```

Comment:
Probably want to `useMemo` this so we don't have to do the string replace on every render.

Reply:
wouldn't `useMemo` require more cycles (dunno what it does but assume some kind of hashing + caching) than a simple replace in under 10 characters string?

Reply:
Potentially.. the thinking here is that it's better to do the calculation once rather than every single time the component function is called to check if it should re-render (even if nothing changes when it does). React function components are called a LOT (anytime any of its parent's get called to check if they need to re-render at a minimum). So my theory/perception here is that even if the memo takes slightly longer once, it's probably going to be net-'cheaper' than doing the replace every single time the component is called.

Reply:
gone now. IMHO can't be simpler now - only aesthetically inspired folks needing separate xml/html markup should complain ;-)